test/integration/data-types.test.js

```diff
@@ -226,7 +277,8 @@ describe(Support.getTestDialectTeaser('DataTypes'), () => {
     }
   });
 
-  it('should handle JS BigInt type', async function() {
+  // Node-oracledb doesn't support JS BigInt yet
```

Comment:
It should still work, the dialect needs to convert the value to string if the connector doesn't support bigint

Reply:
Fixed by fetching BigInt as string